src/binding/python/openpmd_api/scipp/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""openpmd_scipp: A Python package for loading openPMD datasets
+    into scipp DataArrays.
+
+See README.md for documentation
+
+Author:
+    Pawel Ordyna <p.ordyna@hzdr.de>
+
+License:
+GPL - 3.0 license. See LICENSE file for details.
+"""
```

Comment:
@pordyna for openPMD-api, we use the LGPL license. Please confirm we can change the license for files in this PR and then please update the headers accordingly @franzpoeschel .

Reply:
yes sure go for it!